src/test/java/kitchenpos/application/MenuServiceTest.java

```diff
@@ -0,0 +1,457 @@
+package kitchenpos.application;
+
+import jakarta.transaction.Transactional;
+import kitchenpos.domain.*;
+import kitchenpos.infra.PurgomalumClient;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.UUID;
+
+@SpringBootTest
+@Transactional
+class MenuServiceTest {
+    @Autowired
+    private MenuService menuService;
+    @Autowired
+    private MenuRepository menuRepository;
+    @Autowired
+    private MenuGroupRepository menuGroupRepository;
+    @Autowired
+    private ProductRepository productRepository;
+    @Autowired
+    private PurgomalumClient purgomalumClient;
+
+    private MenuGroup menuGroup;
+    private Product product;
+
+    @BeforeEach
+    void setUp() {
+        menuRepository.deleteAll();
+        menuGroupRepository.deleteAll();
+        productRepository.deleteAll();
+
+        menuGroup = new MenuGroup();
+        menuGroup.setId(UUID.randomUUID());
+        menuGroup.setName("한마리메뉴");
+        menuGroupRepository.save(menuGroup);
+
+        product = new Product();
+        product.setId(UUID.randomUUID());
+        product.setName("후라이드");
+        product.setPrice(BigDecimal.valueOf(16000));
+        productRepository.save(product);
+
+    }
+    @DisplayName("메뉴를 생성 할 수 있다.")
+    @Test
+    void create() {
+        // given
+        MenuProduct menuProduct1 = new MenuProduct();
+        menuProduct1.setProductId(product.getId());
+        menuProduct1.setProduct(product);
+        menuProduct1.setQuantity(1);
+
+        List<MenuProduct> menuProducts = List.of(menuProduct1);
+
+        Menu request = new Menu();
+        request.setId(UUID.randomUUID());
+        request.setDisplayed(true);
+        request.setName("후라이드");
+        request.setPrice(BigDecimal.valueOf(16000));
+        request.setMenuGroupId(menuGroup.getId());
+        request.setMenuProducts(menuProducts);
+
+        //when
+        Menu result = menuService.create(request);
+
+        //then
+        Assertions.assertThat(result).isNotNull();
+        Assertions.assertThat(result.getId()).isNotNull();
+        Assertions.assertThat(result.getName()).isEqualTo("후라이드");
+        Assertions.assertThat(result.getPrice()).isEqualTo(BigDecimal.valueOf(16000));
+        Assertions.assertThat(result.getMenuProducts()).hasSize(1);
+
+    }
+
+    @DisplayName("메뉴 생성 시 메뉴 가격이 null 이면 IllegalArgumentException 예외 처리를 한다.")
+    @Test
+    void canNotCreateMenuIfMenuPriceIsNull() {
```

Comment:
테스트 코드의함수명을 짓는것에 대해 어려움을 겪으셨다고 남겨주셨는데 개인적으로 테스트 코드를 작성할때 가독성을 위해 함수명을 한글로 짓기도 합니다 😄 
```suggestion
    void `메뉴생성_실패_가격_Null`() {
```